styles/styles.css

```diff
@@ -157,8 +158,18 @@ body.no-scroll {
   overflow: hidden;
 }
 
-main .section .default-content-wrapper { /* mobile */
-  padding: 0 var(--body-padding);
+.section {
+  width: 100%;
+  position: relative;
+  padding: 0;
+}
+
+.wrapper {
```

Comment:
Do we need this ? Seems that it should be enough to just set the padding, max-width and margin on section.
Are there any issues without adding `.wrapper`  ?